### DISCUSSION TITLE
Linux Installation Defaults to an Old Version of NeoVim

### DISCUSSION THREAD

**Discussion by Harish-Shankar** [neovim/neovim#27374] [Q&A]
### Problem

I was trying to set up kickstart on my schools ssh terminal and upon creating the init.lua file I get the following error

## Error Message
Error detected while processing /nfs/elsrv4/users2/grad/name/.config/nvim/init.lua:
lazy.nvim requires Neovim >= 0.8.0
E5113: Error while calling lua chunk: /nfs/elsrv4/users2/grad/name/.config/nvim/init.lua:358: module '
telescope' not found:
no field package.preload['telescope']
no file './telescope.lua'
no file '/usr/share/luajit-2.1.0-beta3/telescope.lua'
no file '/usr/local/share/lua/5.1/telescope.lua'
no file '/usr/local/share/lua/5.1/telescope/init.lua'
no file '/usr/share/lua/5.1/telescope.lua'
no file '/usr/share/lua/5.1/telescope/init.lua'
no file './telescope.so'
no file '/usr/local/lib/lua/5.1/telescope.so'
no file '/usr/lib/x86_64-linux-gnu/lua/5.1/telescope.so'
no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
[C]: in function 'require'
/nfs/elsrv4/users2/grad/name/.config/nvim/init.lua:358: in main chunk

### Steps to reproduce

PRETTY_NAME="Ubuntu 22.04.3 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.3 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy

NVIM v0.7.2
Build type: Release
LuaJIT 2.1.0-beta3

git clone https://github.com/nvim-lua/kickstart.nvim.git "${XDG_CONFIG_HOME:-$HOME/.config}"/nvim

### Expected behavior

Running the command `nvim` should work and not result in an error

### Neovim version (nvim -v)

v0.7.2

### Vim (not Nvim) behaves the same?

no, vim 8.2.2121

### Operating system/version

Ubuntu 22.04.3 LTS

### Terminal name/version

OpenSSH_8.9p1 Ubuntu-3ubuntu0.6, OpenSSL 3.0.2 15 Mar 2022

### $TERM environment variable

xterm-256color

### Installation

appimage

**Comment by justinmk** [comment]
We do not maintain the ubuntu package.

**Reply by Harish-Shankar** [reply]
I used the appimage because the ubuntu package requires sudo which I do not have permission for.

**Reply by justinmk** [reply]
So... what is your question? The appimage provides the latest version.

**Reply by Harish-Shankar** [reply]
When I run the commands provided, it does not install the latest version. It installs v0.7.2

**Reply by justinmk** [reply]
We don't maintain kickstart.nvim

**Reply by Harish-Shankar** [reply]
I get that but when I followed the instructions on the `Install.md` for Linux using the `.appimage` the version of neovim is v0.7.2. I can't use the ubuntu specific one or build from source as they both require sudo privileges which I do not have my school's lunux server

**Reply by clason** [reply]
I'm sorry, but I have a hard time believing that. If you download the latest stable app image from Github, it will give you the latest release (0.9.5). Distro installs are another matter outside of our control.

**Comment by Harish-Shankar** [comment]
I figured it out but for some reason `neovim` command would open a version of neovim I never installed (0.7.2), but when I would run the command `./nvim.appimage` it would open the version 0.9.5. I created another alias to avoid the problem

**Reply by tbush5** [reply]
I believe you can also move the nvim.appimage app to the /usr/bin/nvim folder so that it finds that version when you run the regular nvim command instead of setting up an alias.
